Team2-Signoff-Communication Application & Display System

jax-commits opened the pull request:
Requesting Signoff for Communication Application & Display System

jax-commits committed:
Create Communication Application & Display System - Signoff.md

JesseTNRoberts reviewed (commented):
I have no idea what the purpose of this subsystem is. it is not clear.

jax-commits committed:
Update Communication Application & Display System - Signoff.md

JesseTNRoberts reviewed (commented):
This has nothing connected to it?

Change the constraints to a table. Where do the constraints come from? 

How will we know that the communication system works for all the sensors?

jax-commits committed:
Update Communication Application & Display System - Signoff.md

JesseTNRoberts reviewed (commented):
I don't understand what this design is for. The explanation of its function needs to be more clear.

jax-commits committed:
Update Communication Application & Display System - Signoff.md

I have updated the top section to include what this subsystem is and why it is needed. It is used to send the information from the PLC to an Arduino to display properly on a screen for the greenhouse employees to see.

JesseTNRoberts reviewed (commented):
I don't understand what this is supposed to do. The description is unclear. 

I gather some sort of gui? This design does not provide the needed function if that is the case and does not have any of the appropriate constraints.

jax-commits committed:
Update Communication Application & Display System - Signoff.md
